python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -579,7 +609,9 @@ def measureCrosstalkCoefficients(self, ratios, ordering, rejIter, rejSigma):
             if ii == jj:
                 values = [0.0]
             else:
-                values = np.array(ratios[ordering[ii]][ordering[jj]])
+                # ratios is ratios[Target][Source]
+                # use jj for Target, use ii for Source, to match ip_isr.
```

Comment:
Slightly beyond the scope, but using indices like `source_idx` and `target_idx`, or even something like `ss` and `tt` instead of `ii` and `jj` could avoid a lot of confusion overall.

Reply:
This is such an obvious improvement that simply never occurred to me.  I've made this change here and in `ip_isr`, so this won't confuse people in the future.